doc/dev_guide/building_from_source.rst

```diff
@@ -1,283 +1,264 @@
+..  code-block:: console
 
-       * `pyyaml <https://pypi.python.org/pypi/PyYAML>`_ version 3.10
-       * `argparse <https://pypi.python.org/pypi/argparse>`_ version 1.1
-       * `msgpack-python <https://pypi.python.org/pypi/msgpack-python>`_ version 0.4.6
-       * `gevent <https://pypi.python.org/pypi/gevent>`_ version 1.1.2
-       * `six <https://pypi.python.org/pypi/six>`_ version 1.8.0
+    $ apt-get install -y git build-essential cmake make zlib1g-dev \
+      libreadline-dev libncurses5-dev libssl-dev libunwind-dev libicu-dev \
+      python3 python3-yaml python3-six python3-gevent
 
-   To install all required dependencies, follow the instructions for your OS:
+    $ git clone https://github.com/tarantool/tarantool.git --recursive
 
-   * For Debian/Ubuntu, say:
+    $ cd tarantool
 
-     .. code-block:: console
+    $ git submodule update --init --recursive
 
-        $ apt install -y build-essential cmake make coreutils sed \
-              autoconf automake libtool zlib1g-dev \
-              libreadline-dev libncurses5-dev libssl-dev \
-              libunwind-dev libicu-dev \
-              python python-pip python-setuptools python-dev \
-              python-msgpack python-yaml python-argparse python-six python-gevent
+    $ make clean         # unnecessary, added for good luck
+    $ rm CMakeCache.txt  # unnecessary, added for good luck
 
-   * For RHEL/CentOS (versions under 8)/Fedora, say:
+    $ mkdir build && cd build
 
-     .. code-block:: console
+    $ # start initiating with build type=RelWithDebInfo
+    $ cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo
 
-         $ yum install -y gcc gcc-c++ cmake make coreutils sed \
-               autoconf automake libtool zlib-devel \
-               readline-devel ncurses-devel openssl-devel \
-               libunwind-devel libicu-devel \
-               python python-pip python-setuptools python-devel \
-               python-msgpack python-yaml python-argparse python-six python-gevent
+    $ make
 
-   * For CentOS 8, say:
+    $ make test
 
-     .. code-block:: console
+.. _building_from_source-fedora:
 
-         $ yum install epel-release
-         $ curl -s https://packagecloud.io/install/repositories/packpack/backports/script.rpm.sh | sudo bash
-         $ yum install -y gcc gcc-c++ cmake make coreutils sed \
-               autoconf automake libtool zlib-devel \
-               readline-devel ncurses-devel openssl-devel \
-               libunwind-devel libicu-devel \
-               python2 python2-pip python2-setuptools python2-devel \
-               python2-yaml python2-six
+Fedora
+~~~~~~
 
-   * For Mac OS X (instructions below are for OS X El Capitan):
+..  code-block:: console
 
-     If you're using Homebrew as your package manager, say:
+    $ yum install -y git gcc gcc-c++ cmake make readline-devel ncurses-devel \
+      openssl-devel zlib-devel libunwind-devel libicu-devel \
+      python3-yaml python3-six python3-gevent
```

Comment:
Hm, it is curious. I didn't know that `python3-pyyaml` has the `python3-yaml` alias. However I suggest to use `python3-pyyaml` for consistency with our RPM package [dependencies](https://github.com/tarantool/tarantool/blob/8b3ec7b928a02a8cc44aca1911cc5d665cf5133d/rpm/tarantool.spec#L98) (so, if we'll find a problem on our side, we'll report it to the documentation team).